lib/watcherManager.js

```diff
@@ -8,10 +8,10 @@ function WatcherManager() {
 	this.directoryWatchers = {};
 }
 
-WatcherManager.prototype.getDirectoryWatcher = function(directory) {
+WatcherManager.prototype.getDirectoryWatcher = function(directory, options) {
 	var DirectoryWatcher = require("./DirectoryWatcher");
 	if(!this.directoryWatchers[directory]) {
```

Comment:
Please separate watchers with different options from each other.

``` js
var key = directory + " " + JSON.stringify(options);
this.directoryWatchers[key]
```

Elsewise this would cause issues:

``` js
watchFile("/path/a", null, { usePolling: true });
watchFile("/path/b", null, { usePolling: false }); // options would be ignored as there is already a DirectoryWatcher
```